Source/Virtusize.swift

```diff
@@ -28,7 +28,12 @@ public class Virtusize {
 	public static var APIKey: String?
 	public static var userID: String?
 	public static var environment = VirtusizeEnvironment.global
+    // Default to the language set fot the phone
     public static var language: String = Locale.preferredLanguages[0]
```

Comment:
Do we use App language? If we're using device language, code should be
```
NSLocale.current.languageCode
```

Reply:
The application is in Swift, `NSLocale` is bridged by `Locale` since Swift 4.1

```
Important
The Swift overlay to the Foundation framework provides the Locale structure, which bridges to the NSLocale class. For more information about value types, see Working with Cocoa Frameworks in Using Swift with Cocoa and Objective-C (Swift 4.1).
```
https://developer.apple.com/documentation/foundation/nslocale

Reply:
Oh, Thanks!! I used swift3